include/boost/redis/connection.hpp

```diff
@@ -755,25 +825,35 @@ class basic_connection {
       mpx_.cancel_on_conn_lost();
    }
 
-   template <class, class> friend struct detail::reader_op;
-   template <class, class> friend struct detail::writer_op;
-   template <class> friend struct detail::exec_op;
-   template <class, class> friend class detail::run_op;
+   // Used by both this class and connection
+   void set_run_logger(const logger& lgr, const config& cfg)
```

Comment:
Shouldn't this be named `set_logger` since the logger can also be used in `async_exec`?